alibi_detect/saving/saving.py

```diff
@@ -123,9 +124,9 @@ def _save_detector_config(detector: ConfigurableDetector, filepath: Union[str, o
         File path to save serialized artefacts to.
     """
     # Get backend, input_shape and detector_name
-    backend = detector.meta.get('backend', 'tensorflow')
-    if backend != 'tensorflow':
-        raise NotImplementedError("Currently, saving is only supported with backend='tensorflow'.")
+    backend = detector.meta.get('backend', None)
+    if backend not in (None, 'tensorflow', 'sklearn'):
```

Comment:
`None` is now included since detectors such as `KSDrift` now have no `backend` in their config.